### PR TITLE
PSMDB-1401 Release notes 4.2.25-25

### PR DESCRIPTION
**Pull request by nastena1606**
* Added Telemetry documentation

modified:   .github/workflows/main.yml
	modified:   .gitignore
	modified:   docs/404.md
	modified:   docs/css/percona.css
	modified:   docs/install/apt.md
	modified:   docs/install/docker.md
	modified:   docs/install/index.md
	modified:   docs/install/yum.md
	new file:   docs/release_notes/4.2.25-25.md
	modified:   docs/release_notes/index.md
	new file:   docs/telemetry.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release_notes/4.2.25-25.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* [SERVER-80488](https://jira.mongodb.org/browse/SERVER-80488) - Avoided traversing routing table in balancer split chunk policy
+
+* [SERVER-81966](https://jira.mongodb.org/browse/SERVER-81966) - Fixed the issue that caused the modification of the original ChunkMap vector during the chunk migration and that could lead to data loss. The issue affects MongoDB versions 4.4.25, 5.0.21, 6.0.10 through 6.0.11 and 7.0.1 through 7.0.2. Requires stopping all chunk merge activities and restarting all the binaries in the cluster (both `mongod` and `mongos`). 
+
+Find the full list of bug fixes in the [MongoDB 4.2.25 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/4.2/#4.2.25---dec-6--2023)
+
+## New Features
+
+* [PSMDB-1241](https://jira.percona.com/browse/PSMDB-1241) - Implement the `connectRetries` and the `connectTimeoutMS` configuration file options
```
Comment (igorsol) [inline docs/release_notes/4.2.25-25.md]: PSMDB-1241 was not ported to v4.2 branch. This should be removed.
@ktrushin correct me if I'm wrong

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by janwieremjewicz — APPROVED** [review]
Other than the comment already added, it's ok

File: docs/release_notes/4.2.25-25.md (hunk 1 above)
Comment (nastena1606) [reply docs/release_notes/4.2.25-25.md]: Fixed

**Review by igorsol — APPROVED** [review]

File: docs/release_notes/4.2.25-25.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Percona Server for MongoDB 4.2.25-25 (2024-02-)
```
Comment (sandraromanchenko) [inline docs/release_notes/4.2.25-25.md]:
```suggestion
# Percona Server for MongoDB 4.2.25-25 (2024-02-08)
```

File: mkdocs-base.yml [hunk 3]
```diff
@@ -113,7 +113,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-4.2.pdf'
           cover_title: 'Server for MongoDB Documentation 4.2'
-          cover_subtitle: 4.2.24-24 (March 9, 2023)
+          cover_subtitle: 4.2.25-25 (January , 2024)
```
Comment (sandraromanchenko) [inline mkdocs-base.yml]:
```suggestion
          cover_subtitle: 4.2.25-25 (February 8, 2024)
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**